behaviors/text.js

```diff
@@ -121,6 +138,36 @@ module.exports = function (result, args) {
       </label>
     `);
 
+  // if it's a date input, init datepicker for non-native browsers
+  if (_.includes(['datetime-local', 'date', 'time'], type)) {
+    result.binders.datepicker = {
+      publish: true,
+      bind: function (el) {
+        var observer = this.observer;
+
+        // when instantiating, convert from the ISO format (what we save) to firefox's format (what the datepicker needs)
+        if (!datepicker.hasNativePicker()) {
+          el.value = moment(observer.value()).format(firefoxDateFormat);
+          datepicker.init(el, { onChange: function (date) {
+            // when the datepicker changes, convert it back to ISO format
+            observer.setValue(moment(date).utc().format());
```

Comment:
Should this be `.format(defaultDateFormat)`? Or is there another format that would belong in there? (moment.ISO_8601?)